app/src/main/java/ch/epfl/sdp/blindwar/domain/game/Game.kt

```diff
@@ -13,27 +13,22 @@ import ch.epfl.sdp.blindwar.data.music.MusicMetadata
  */
 abstract class Game(
     gameInstance: GameInstance,
-    protected val assetManager: AssetManager,
     protected val context: Context
 ) {
     /** Encapsulates the characteristics of a game instead of its logic **/
-    private val game: GameInstance = gameInstance
+    protected val game: GameInstance = gameInstance
 
-    protected lateinit var musicSound: MusicSound
+    protected lateinit var musicController: MusicController
 
     private val gameParameter: GameParameter = gameInstance
         .gameConfig
         .parameter
 
-    private val gameDifficulty: GameDifficulty = gameInstance
-        .gameConfig
-        .difficulty
-
     /** Player game score **/
     var score = 0
         protected set
 
-    private var round = 0
+    var round = 0
```

Comment:
Protected set is better I think here. It would be more consistent with the score "attribute"